scripts/init.sh

```diff
@@ -70,9 +70,21 @@ $aws s3 cp "$WDL_WORKFLOW_URI" .
 $aws s3 cp "$WDL_INPUT_URI" wdl_input.json
 
 handle_error() {
+  # Add enhanced logging for our most common termination types
+  EXIT_CODE=$?
+  if [[ $EXIT_CODE == 137 ]]; then
+    echo ERROR: container terminated with SIGKILL, this is most likely because memory usage was above container limits >> /dev/stderr
+    exit $EXIT_CODE
+  fi
+
+  if [[ $EXIT_CODE == 143 ]]; then
+    echo ERROR: container terminated with SIGTERM, this is most likely because of a spot instance termination or a timeout >> /dev/stderr
+    exit $EXIT_CODE
+  fi
+
   OF=wdl_output.json;
   EP=.cause.stderr_file;
-  if jq -re .error $OF; then
+  if jq -re .error $OF 2> /dev/null; then
```

Comment:
Would we expect there to be errors thrown here given the above code?

Reply:
Good point, there are theoretically other sources of failure, though unlikely, so I think it is best to leave it